lib/elixir/lib/gen_server.ex

```diff
@@ -583,41 +592,48 @@ defmodule GenServer do
       def handle_call(msg, _from, state) do
         proc =
           case Process.info(self(), :registered_name) do
-            {_, []}   -> self()
+            {_, []} -> self()
             {_, name} -> name
           end
 
         # We do this to trick Dialyzer to not complain about non-local returns.
         case :erlang.phash2(1, 1) do
-          0 -> raise "attempted to call GenServer #{inspect proc} but no handle_call/3 clause was provided"
-          1 -> {:stop, {:bad_call, msg}, state}
+          0 ->
+            raise "attempted to call GenServer #{inspect(proc)} but no handle_call/3 clause was provided"
+
+          1 ->
+            {:stop, {:bad_call, msg}, state}
         end
       end
 
       @doc false
       def handle_info(msg, state) do
         proc =
           case Process.info(self(), :registered_name) do
-            {_, []}   -> self()
+            {_, []} -> self()
             {_, name} -> name
           end
-        :error_logger.error_msg('~p ~p received unexpected message in handle_info/2: ~p~n',
-                                [__MODULE__, proc, msg])
+
+        pattern = '~p ~p received unexpected message in handle_info/2: ~p~n'
```

Comment:
Could/should this be stored in a module attribute?

Reply:
It could but this way is completely fine.